src/spaces/continuous.jl

```diff
@@ -134,9 +130,8 @@ end
 # move the agent in the GridSpace; only change its position field
 function move_agent!(agent::A, pos::ValidPos, model::ABM{<:ContinuousSpace{D},A}
     ) where {D, A<:AbstractAgent}
-    for i in 1:D
-        pos[i] > spacesize(model)[i] && error("position is outside space extent!")
-    end
+    space_size = spacesize(model)
+    all(i -> 0 <= pos[i] < space_size[i], 1:D) || error("position is outside space extent!")
```

Comment:
it is more efficient here to use `any` and `&&` instead of `all` and `||`. This will make the clause exit as soon as possible.

Reply:
I think it's actually the same, `all` will return false the first time a pos coordinate is not inside extent, with `any` it would return true the first time a pos coordinate is not inside extent, so with exchanging the logical operator everything would work the same